data-truth/nytimes/nytimes.py

```diff
@@ -8,15 +8,15 @@
 RELATIVE_PATH = "data-truth/nytimes"
 
 # Download raw data
-us = pd.read_csv(US_URL, dtype={'cases': int, 'deaths': int}).fillna(value = 'NA')
+us = pd.read_csv(US_URL).dropna().astype(dtype={'cases': int, 'deaths': int})
```

Comment:
Wouldn't this drop rows that have NA in any column? I think there are NAs in both the FIPS code column and the deaths count column, but do we want to drop them? As those rows still have information of cases counts and the state associated with it, so maybe other apps like the shiny app are still using those information to do some aggregation. Maybe you can check with @jarad on this?

Reply:
Yes, that is correct; it would drop any row with NA values. And that there are NAs in FIPS as well as deaths. NAs in either the deaths are due to the Puerto Rico issue. The NAs in the FIPS code are aggregations of multiple FIPS codes to a city (Like New York City) which we don't use in any of our visualizations or forecasts AFAIK. 

But yes, I will confirm with Jarad before this change is made.

Reply:
Confirmed with @jarad that this should not be a problem.